kale/prepdata/tabular_transform.py

```diff
@@ -0,0 +1,71 @@
+from typing import Any, Optional
```

Comment:
Missing doc at the top.

Reply:
I have added a docstring at the top of the file